user_manual/index.rst

```diff
@@ -1,8 +1,8 @@
 .. _index:
 
-============================================
-Nextcloud |version| user manual introduction
```

Comment:
This is shipped as PDF file with the release, that's why we want to have the version number in there, iirc

Reply:
I don't see the PDF (at least in v27 and v28). The help page loads the HTML doc.

The version is still in the PDF - on the title page for example. It's just that now instead of always saying `latest` (including above) no matter what version of the doc someone is looking at - it'll say the actual version.

I can see putting it back on that particular page maybe in the first paragraph or something. But it's weird IMO to have it embedded in the ToC entry for the Introduction Chapter (even if we do make it the correct version). I can see having it in place for the "What's new" chapter maybe.

I'll re-look a bit more at it maybe tomorrow. Already took longer than planned since I hadn't messed with the PDF builds before. They're... a little weird. :)